pynxtools/dataconverter/readers/em/reader.py

```diff
@@ -0,0 +1,185 @@
+        # subparser = NxEmNxsMTexSubParser(entry_id, file_paths[0])
+        # subparser.parse(template)
+        # TODO::check correct loop through!
+
+        # add further with resolving cases
+        # if file_path is an HDF5 will use hfive parser
+        # sub_parser = "nxs_pyxem"
+        # subparser = NxEmNxsPyxemSubParser(entry_id, file_paths[0])
+        # subparser.parse(template)
+        # TODO::check correct loop through!
+
+        # sub_parser = "image_tiff"
+        # subparser = NxEmImagesSubParser(entry_id, file_paths[0])
+        # subparser.parse(template)
+        # TODO::check correct loop through!
+
+        # sub_parser = "zipped_nion_project"
+        # subparser = NxEmZippedNionProjectSubParser(entry_id, file_paths[0])
+        # subparser.parse(template, verbose=True)
+        # TODO::check correct loop through!
+
+        # sub_parser = "velox_emd"
+        subparser = RsciioVeloxSubParser(entry_id, file_paths[0], verbose=False)
+        subparser.parse(template)
+
+        # for dat_instance in case.dat_parser_type:
+        #     print(f"Process pieces of information in {dat_instance} tech partner file...")
+        #    continue
+        #    # elif case.dat_parser_type == "zip":
+        #    #     zip_parser = NxEmOmZipEbsdParser(case.dat[0], entry_id)
+        #    #     zip_parser.parse(template)
+        #    # elif case.dat_parser_type == "dream3d":
+        #    #     dream_parser = NxEmOmDreamThreedEbsdParser(case.dat[0], entry_id)
+        #    #     dream_parser.parse(template)
+        #    # elif case.dat_parser_type == "kikuchipy":
+        #    # elif case.dat_parser_type == "pyxem":
+        #    # elif case.dat_parser_type == "score":
+        #    # elif case.dat_parser_type == "qube":
+        #    # elif case.dat_parser_type == "paradis":
+        #    # elif case.dat_parser_type == "brinckmann":
+        # at this point the data for the default plots should already exist
+        # we only need to decorate the template to point to the mandatory ROI overview
+        # print("Create NeXus default plottable data...")
+        # em_default_plot_generator(template, 1)
+
+        run_block = False
+        if run_block is True:
+            nxs_plt = NxEmDefaultPlotResolver()
+            # if nxs_mtex is the sub-parser
+            resolved_path = nxs_plt.nxs_mtex_get_nxpath_to_default_plot(
+                entry_id, file_paths[0])
+            # print(f"DEFAULT PLOT IS {resolved_path}")
+            if resolved_path != "":
+                nxs_plt.annotate_default_plot(template, resolved_path)
+
+        debugging = False
+        if debugging is True:
+            print("Reporting state of template before passing to HDF5 writing...")
+            for keyword in template.keys():
+                print(keyword)
+                # print(type(template[keyword]))
+                print(template[keyword])
+
+        print("Forward instantiated template to the NXS writer...")
+        return template
```

Comment:
Unused code.

Reply:
unused because was deactivated intentionally for debugging purposes, cleaned in pynxtools-em